src/Writer/Common/Entity/Worksheet.php

```diff
@@ -92,4 +99,17 @@ public function getId(): int
         // sheet index is zero-based, while ID is 1-based
         return $this->externalSheet->getIndex() + 1;
     }
+
+    public function getAutoFilter(): AutoFilter
```

Comment:
This API can be moved to `Sheet` which already holds configurations for the actual sheets